packages/office-ui-fabric-react/src/components/Layer/Layer.types.ts

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
-import { LayerBase } from './Layer.base';
+import { PortalLayerBase } from './PortalLayer.base';
```

Comment:
should we rename the file back to Layer.base?

Reply:
Hmm.. I can think of some pros/cons both ways. I'll confirm with @dzearing to see where he leans. I'm guessing we probably should.